src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:31945/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:31945/project/issues?id=sample&branch=feature3&issues=AWD5c7ec1MwIza6SkTy7&open=AWD5c7ec1MwIza6SkTy7)

[ISSUE_KEY]: AWD5c7ec1MwIza6SkTy7